piq/vsi.py

```diff
@@ -0,0 +1,285 @@
+r"""Implemetation of Visual Saliency-induced Index
+Code is based on MATLAB version for computations in pixel domain
+https://sse.tongji.edu.cn/linzhang/IQA/VSI/VSI.htm
+
+References:
+    https://ieeexplore.ieee.org/document/6873260
+"""
+
+import torch
+from torch.nn.modules.loss import _Loss
+from torch.nn.functional import conv2d, avg_pool2d, interpolate
+from .utils import _validate_input, _adjust_dimensions
+
+
+def vsi(prediction: torch.Tensor, target: torch.Tensor, reduction: str = 'mean', data_range: float = 1.,
```

Comment:
Since `data_range` can be either `int` or `float`, correct data type will be:
```suggestion
def vsi(prediction: torch.Tensor, target: torch.Tensor, reduction: str = 'mean', data_range: Union[int, float] = 1.,
```